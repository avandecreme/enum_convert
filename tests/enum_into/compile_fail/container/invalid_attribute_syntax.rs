use enum_convert::EnumInto;

#[derive(EnumInto)]
#[enum_into(Target1; Target2)] // semi colon instead of comma
enum Source {
    Unit,
}

enum Target1 {
    Unit,
}

enum Target2 {
    Unit,
}

fn main() {}
