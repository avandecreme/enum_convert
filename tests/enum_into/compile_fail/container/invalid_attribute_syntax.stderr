error: expected `,`
 --> tests/enum_into/compile_fail/container/invalid_attribute_syntax.rs:4:20
  |
4 | #[enum_into(Target1; Target2)] // semi colon instead of comma
  |                    ^
